### PR TITLE
TGP-457 - add homepage and cya redirects to homepage

### PR DESCRIPTION
**Pull request by ChrisCookOC**

### CODE REVIEW

File: app/controllers/CheckYourAnswersController.scala [hunk 1]
```diff
@@ -60,14 +60,14 @@ class CheckYourAnswersController @Inject()(
...
-      val (maybeErrors, maybeModel) = TraderProfile.build(request.userAnswers).pad
+      val (maybeErrors, maybeModel) = TraderProfile.build(request.userAnswers, request.eori).pad
 
       val errors = maybeErrors.map { errors =>
         errors.toChain.toList.map(_.path).mkString(", ")
       }.getOrElse("")
 
       maybeModel.map { model =>
-        routerConnector.submitTraderProfile(model).map { _ =>
+        routerConnector.submitTraderProfile(model, request.eori).map { _ =>
```
Comment (liandra-giacomine) [inline app/controllers/CheckYourAnswersController.scala:70]: we're not handling a failed future here

**Review by liandra-giacomine — APPROVED** [review]
can we also create a PR to update app-config-base?

**Review by liandra-giacomine — CHANGES_REQUESTED** [review]
Can we add a test case for when the connector call fails?

**Comment by ChrisCookOC** [comment]
> can we also create a PR to update app-config-base?

What do we need to update there?

**Comment by ChrisCookOC** [comment]
> Can we add a test case for when the connector call fails?

Added

**Review by Ewan-James-Donovan — DISMISSED** [review]

**Comment by ChrisCookOC** [comment]
> > can we also create a PR to update app-config-base?
> 
> What do we need to update there?

Done and merged :)

**Review by liandra-giacomine — APPROVED** [review]

**Review by Ewan-James-Donovan — APPROVED** [review]

**Review by mycp98 — APPROVED** [review]

File: test/controllers/CheckYourAnswersControllerSpec.scala [hunk 2]
```diff
@@ -182,11 +183,37 @@ class CheckYourAnswersControllerSpec extends SpecBase with SummaryListFluency wi
...
+        running(application) {
+          val request = FakeRequest(POST, routes.CheckYourAnswersController.onPageLoad.url)
+
+          intercept[RuntimeException] {
+            await(route(application, request).value)
           }
         }
       }
 
+
```
Comment (besscerule) [inline test/controllers/CheckYourAnswersControllerSpec.scala:216]: remove line

**Review by besscerule — APPROVED** [review]

**Pull request merged**